client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
+"use strict";
+
+self.addEventListener("push", function(event) {
```

Comment:
So far, sure. But it'll be very useful when we need to add other things to the service worker (offline support) and we'll get the benefits of transpiling just now.